cmd/revad/svcs/httpsvcs/handlers/auth/auth.go

```diff
@@ -145,6 +158,9 @@ func New(m map[string]interface{}) (httpserver.Middleware, int, error) {
 			// store user and core access token in context.
 			ctx := user.ContextSetUser(r.Context(), u)
 			ctx = token.ContextSetToken(ctx, tkn)
+
+			ctx = metadata.AppendToOutgoingContext(ctx, "x-access-token", tkn) // FIXME hardcoded metadata key. use  PerRPCCredentials?
```

Comment:
@labkode I think you were missing something crucial with this ;-)

Reply:
Give it a try, I remember having problems using the context directly like that, some RPC error because the metadata was not correctly created

Reply:
oh this definitely works